src/legacy/core_plugins/kibana/public/dashboard/index.js

```diff
@@ -70,9 +70,12 @@ uiRoutes
       $scope.find = (search) => {
         return services.dashboards.find(search, $scope.listingLimit);
       };
-      $scope.edit = ({ id }) => {
+      $scope.editItem = ({ id }) => {
         kbnUrl.redirect(createDashboardEditUrl(id));
       };
+      $scope.getEditUrl = ({ id }) => {
+        return `#${createDashboardEditUrl(id)}`;
```

Comment:
This would need to be wrapped in `chrome.addBasePath` to make sure it's working behind a reverse proxy (like in our dev environment).

Reply:
done